cgi-bin/LJ/Console/Command/EmailAlias.pm

```diff
@@ -68,12 +71,28 @@ sub execute {
         $dbh->do("REPLACE INTO email_aliases VALUES (?, ?)", undef, $alias, $value);
         return $self->error("Database error: " . $dbh->errstr)
             if $dbh->err;
+        if ( $alias ne $alias_with_dashes ) {
+            $dbh->do( "REPLACE INTO email_aliases VALUES (?, ?)",
```

Comment:
Hmm, a thought, if a user renames, and goes from, e.g, `abc_123` to `abc`, they'll have previously had two entries in the db:

abc-123 => email@example.com
abc_123 => email@example.com

On a rename, they only have "abc". Does that then let the old "abc-123" alias hang around?

Note: this isn't an issue with renames because that does do a delete_email_alias if we're not redirecting emails.